File: uCookApp/uCookApp/Models/ToDo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace uCookApp.Models
{
    public class ToDo : BaseItem
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Summary { get; set; }
        public DateTime DateCompleted { get; set; }

        public string ToJson()
        {
            //Replace with an actual ToJson method
            return Newtonsoft.Json.JsonConvert.SerializeObject(this);
        }
    }
}

File: Client/todo-app/src/components/ToDo.js
import React from 'react'

const ToDo = ({ onClick, item }) => (
  <li
    onClick={onClick}
  >
    <p>
      <label><b>Id:</b> </label> {item.Id}|
      <label>   <b>Title:</b> </label> {item.Title}|
      <label>   <b>Summary:</b> </label> {item.Summary}|
      <label>   <b>Date Completed:</b> </label> {item.DateCompleted}|
    </p>
  </li>
)

export default ToDo